Reneiski_Homework1/Reneiski_Homework1/Program.cs

```diff
@@ -1,28 +1,279 @@
-            Belaz.SetWorker(ivan);
-            Belaz.SetWorker(semen);
-
-            Belaz.ShowAllEmployeeInfo();
-            Belaz.ShowEmployeeNameInfo();
-            Belaz.ShowEmployeeAgeInfo();
-            Belaz.ShowEmployeePositionInfo();
-            Belaz.ShowEmployeeNumber();
-            Belaz.ShowEmployeeByPosition();
-            Belaz.ShowEmployeeWork();
+            //Lists for Tasks
+            List<int> ints = new List<int>() { 1, 2, 3, 4, 5, 6, 7, 8, 9, 11 };
+            List<string> strings = new List<string>() { "one", "two", "three", "five", "six", "seven", "eight", "nine", "ten" };
+
+            //SumOfEvens(ints);
+            //PrintWordOfLengthFive(strings);
+            //PrintWordOfLengthLinq(strings);
+            //LinkedListAddTen();
+            //UniteTwoLinkedLists();
+
+            //Напишите метод для нахождения суммы всех четных чисел в списке.
+            //В Main создайте список, содержащий не менее 10 целых чисел, вызовите свой метод и выдите полученный результат.
+
+            static void SumOfEvens(List<int> ints)
+            {
+                int sumOfEvens = 0;
+
+                foreach (var item in ints)
+                {
+                    if (item % 2 == 0)
+                    {
+                        sumOfEvens += item;
+                    }
+                }
+                Console.WriteLine(sumOfEvens);
+            }
+
+            //В Main создайте список строк, где содержатся слова разной длины.
+            //Напишите статический метод для вывода каждого слова из списка, состоящего ровно из 5 букв.
+
+            static void PrintWordOfLengthFive(List<string> strings)
+            {
+                foreach (var item in strings)
+                {
+                    if (item.Length == 5)
+                    {
+                        Console.WriteLine(item);
+                    }
+                }
+            }
+
+            //Измените свой код, чтобы предложить пользователю ввести длину слова для поиска. (через LINQ)
+            static void PrintWordOfLengthLinq(List<string> strings)
+            {
+                Console.WriteLine("Please type the length");
+
+                int lengthFromUser = int.Parse(Console.ReadLine());
+                var lengthOfString = from l in strings where l.Length == lengthFromUser select l;
+
+                foreach (var item in lengthOfString)
+                {
+                    Console.WriteLine(item);
+                }
+            }
+
+            //Создайте LinkedList и два элемента, вставьте второй элемент после каждого вхождения первого элемента в списке.
+            //Итак, если список равен [2,4,3,2,8,2,5,1,2], а элементы равны 2 и 10, результатом будет [2,10,4,3,2,10,8,2,10,5,1,2,10]
+
+            static void LinkedListAddTen()
+            {
+                List<string> ints = new List<string>() { "2", "4", "3", "2", "8", "2", "5", "1", "2" };
+
+                var linkedInts = new LinkedList<string>(ints);
```

Comment:
you may create LinkedList from start with all elements above (no need to use List separately)

Reply:
Fixed